stl/inc/flat_map

```diff
@@ -1206,6 +1183,24 @@ private:
         }
     }
 
+    template <class _KeyTy, class... _MappedArgTypes>
+    pair<iterator, bool> _Try_emplace(_KeyTy&& _Key_val, _MappedArgTypes&&... _Mapped_args) {
+        auto _Key_it = _STD lower_bound(_Data.keys.begin(), _Data.keys.end(), _Key_val, _Key_compare);
+        if (_Key_it != _Data.keys.end() && _Key_equal(*_Key_it, _STD forward<_KeyTy>(_Key_val))) {
+            // Already exists
+            return {this->begin() + _STD distance(_Data.keys.begin(), _Key_it), false};
+        } else {
+            // Need to insert

```

Comment:
Style suggestion: After `if ... return`, we don't need the `else`. Sometimes it can be nice to keep to highlight symmetry, but the cases here are very different, so saving a level of indentation with an early `return` would be nicer.